:sparkles: skeleton

TheNoobProgrammeur opened the pull request:
Mise ne place du Menu de l'application.
Utilisation de la lib fontawesome. Voir si on peut utiliser le ds.

Issue : #16

TheNoobProgrammeur committed:
🚧 🚧 :construction: Add Menu

TheNoobProgrammeur committed:
🚧 🚧 :construction: Update Name menu

TheNoobProgrammeur committed:
🚧 🚧 :construction: Fix user active status

TheNoobProgrammeur committed:
:heavy_plus_sign: adding fontawesome icons

TheNoobProgrammeur committed:
:construction: add fontawesome icons in Menu

acroquelois commented on src/app/shared/layout/menu.component.html:
```diff
@@ -0,0 +1,41 @@
+        >
+      </li>
+      <li>
+        <a [routerLink]="['/register']" routerLinkActive="active"
```
Pq le register dans la navbar ??

acroquelois replied:
Si on se base sur la maquette, il y a une page spécifique pour le login/register

TheNoobProgrammeur replied:
Je peux retirer register, mais je pense que avoir le login dans le menu quand on est pas connecté est intéressent. Surtout qu'un utilisateur non connecter pourra aller sur des pages comme home.

octogene replied:
Hum...Comment un utilisateur non connecté pourrait voir le menu sachant que la page qui s'affiche par défaut pour un utilisateur non connecté est celle du register ? (cf. Landing Page -> Figma). Et j'ai vu nulle part qu'un utilisateur non connecté peut aller sur la home page.

TheNoobProgrammeur replied:
En effet j’ai relu les Must have, je pensais que la page ou les collections était affiché pouvais être visible en non connecter.
Donc pas de menu en non co car on peut rien faire.

acroquelois reviewed (commented):
La logique tu code est bonne pour moi 👍
Par contre le design est trop loin de la maquette

TheNoobProgrammeur commented:
> La logique tu code est bonne pour moi +1
> Par contre le design est trop loin de la maquette

J'ai retoucher les graphisme pour coler mieux a la maquette et j'ai cette version :
![image](https://user-images.githubusercontent.com/34750693/82749048-dca2a500-9da6-11ea-9653-bbf4e9d9b1c0.png)

TheNoobProgrammeur committed:
:lipstick: Harmonisation du SCSS avec la maquette

TheNoobProgrammeur committed:
:lipstick: changement du fichier favicon.ico

TheNoobProgrammeur committed:
:sparkles: add info page + Update Menu

This page it is default page for anonymous visitor.
If User it is login I can see in `/info`

TheNoobProgrammeur committed:
🚧 Insert Info module in anonymous menu

TheNoobProgrammeur committed:
:Removing code or files:  The death of art ... I I will come back

TheNoobProgrammeur committed:
:lipstick:  The death of art ... I I will come back

TheNoobProgrammeur committed:
:lipstick:  The death of art ... I will come back... two ... I forgot files

TheNoobProgrammeur committed:
Merge branch 'feat/skeleton' of https://github.com/enneagone/alexandrie-front into feat/skeleton

TheNoobProgrammeur commented:
@acroquelois @Nahalu  Vous pouvez donner un avie final pour cloture cette issue ?

acroquelois committed:
:twisted_rightwards_arrows: Merge branch 'master' into feat/skeleton

beaussan commented:
:tada: This PR is included in version 1.1.0 :tada:

The release is available on [GitHub release](https://github.com/enneagone/alexandrie-front/releases/tag/v1.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
